### PR TITLE
Use s3 for backups

### PR DESCRIPTION
**Pull request by magikid**
Migrated from PR #432.  See that one for a walk through of what the PR changes and why.

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -11,6 +11,9 @@ export GIT_TOKEN=
 export GOOGLE_API_KEY=
 export GOOGLE_CX=
 export WOLFRAM_ALPHA_ID=
+export OPENAI_API_KEY=
```
Comment (zmattingly) [inline .env-example:14]: We no longer need the OpenAI env var.

**Review by zmattingly — APPROVED** [review]
With the caveat that I have limited devops knowledge, LGTM. 👍

**Comment by zmattingly** [comment]
@magikid Issue with `make db-restore` but maybe I'm missing something?

```
./scripts/db-restore.sh: line 38: s3cmd: command not found
make: *** [db-restore] Error 1
```

**Comment by magikid** [comment]
Notes from chat:

- [ ] add inputs to restore script so we can more easily restore a db from one bot/bucket to another
- [ ] Rather than putting a link to the resource, add a link to the DO UI: https://cloud.digitalocean.com/spaces/lal-brain-backups?path=2024-06-12/

**Pull request merged**